.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": true
+  },
+  "[typescript]": {
```

Comment:
I didn't add `[javascriptreact]` or `[typescriptreact]`, but I assume folks will know to do that if they're working in react-land.